superset-frontend/src/dashboard/components/CrossFilterScopingModal/CrossFilterScopingModal.tsx

```diff
@@ -19,7 +19,7 @@
 import { t } from '@superset-ui/core';
 import React, { FC } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import { StyledModal } from 'src/common/components/Modal';
+import { StyledModal } from 'src/components/Modal';
```

Comment:
This makes my prior comment even more valid, I think... if there's a named `StyledModal` export, we should NOT bring in the default export as `StyledModal`. Anyway...

Reply:
Totally agree. The default export and `StyleModal` have different implementations.